HTAN.model.csv

```diff
@@ -7,13 +7,22 @@ Patient,HTAN patient,,"Component, HTAN Participant ID",,FALSE,Individual Organis
-CDS Other Data Type Specified,Other types of data associated with the content.,,CDS Data Type,,FALSE,Sequencing,,,
+CDS Sequencing Template,"CDS compatible template file, includes attributes for Genomic Reference, Library Layout, Data Type, Sequencing Platform, Library Selection Method",,"Component, Filename, File Format, HTAN Data File ID, HTAN Parent Biospecimen ID, CDS library_id, CDS library_strategy, CDS library_source, CDS library_selection, CDS library_layout, CDS platform, CDS instrument_model, CDS design_description, CDS reference_genome_assembly, CDS custom_assembly_fasta_file_for_alignment, CDS bases, CDS number_of_reads, CDS coverage, CDS avg_read_length, CDS sequence_alignment_software",,TRUE,,,,
+CDS library_id,Short unique identifier for the sequencing library.,,,,TRUE,Sequencing,,https://dataservice.datacommons.cancer.gov/#/resources,str
+CDS library_strategy,Library strategy,"AMPLICON, ATAC-seq, Bisulfite-Seq, ChIA-PET, ChIP-Seq, CLONE, CLONEEND, CTS, DNase-Hypersensitivity, EST, FAIRE-seq, FINISHING, FL-cDNA, Hi-C, MBD-Seq, MeDIP-Seq, miRNA-Seq, MNase-Seq, MRE-Seq, ncRNA-Seq, OTHER, POOLCLONE, RAD-Seq, RIP-Seq, RNA-Seq, SELEX, ssRNA-seq, Synthetic-Long-Read, Targeted-Capture, Tethered Chromatin Conformation Capture, Tn-Seq, WCS, WGA, WGS, WXS",,,TRUE,Sequencing,,https://dataservice.datacommons.cancer.gov/#/resources,str
+CDS library_source,The Library Source specifies the type of source material that is being sequenced,"GENOMIC, GENOMIC SINGLE CELL, METAGENOMIC, METATRANSCRIPTOMIC, OTHER, SYNTHETIC, TRANSCRIPTOMIC, TRANSCRIPTOMIC SINGLE CELL, VIRAL RNA",,,TRUE,Sequencing,,https://dataservice.datacommons.cancer.gov/#/resources,str
+CDS library_selection,Library Selection Method,"5-methylcytidine antibody, CAGE, cDNA, cDNA_oligo_dT, cDNA_randomPriming, CF-H, CF-M, CF-S, CF-T, ChIP, DNAse, HMPR, Hybrid Selection, Inverse rRNA, MBD2 protein methyl-CpG binding domain, MDA, MF, MNase, MSLL, Oligo-dT, other, Padlock probes capture method, PCR, PolyA, RACE, RANDOM, RANDOM PCR, Reduced Representation, repeat fractionation, Restriction Digest, RT-PCR, size fractionation, unspecified",,,TRUE,Sequencing,,https://dataservice.datacommons.cancer.gov/#/resources,str
+CDS library_layout,Paired-end or Single,"Paired-end, Single-end",,,TRUE,Sequencing,,https://dataservice.datacommons.cancer.gov/#/resources,str
+CDS platform,Sequencing Platform used for Sequencing,"LS454, ABI_SOLID, BGISEQ, CAPILLARY, COMPLETE_GENOMICS, HELICOS, ILLUMINA, ION_TORRENT, OXFORD_NANOPORE, PACBIO_SMRT",,,TRUE,Sequencing,,https://dataservice.datacommons.cancer.gov/#/resources,str
+CDS instrument_model,Instrument model used for sequencing,"LS454, ILLUMINA, HELICOS, ABI_SOLID, COMPLETE_GENOMICS, PACBIO_SMRT, ION_TORRENT, CAPILLARY, OXFORD_NANOPORE, BGISEQ",,,TRUE,Sequencing,,https://dataservice.datacommons.cancer.gov/#/resources,str

```

Comment:
Valid values should be

454 GS
454 GS 20
454 GS FLX
454 GS FLX+
454 GS FLX Titanium
454 GS Junior


HiSeq X Five
HiSeq X Ten
Illumina Genome Analyzer
Illumina Genome Analyzer II
Illumina Genome Analyzer IIx
Illumina HiScanSQ
Illumina HiSeq 1000
Illumina HiSeq 1500
Illumina HiSeq 2000
Illumina HiSeq 2500
Illumina HiSeq 3000
Illumina HiSeq 4000
Illumina iSeq 100
Illumina NovaSeq 6000
Illumina MiniSeq
Illumina MiSeq
NextSeq 500
NextSeq 550

Helicos HeliScope

AB 5500 Genetic Analyzer
AB 5500xl Genetic Analyzer
AB 5500x-Wl Genetic Analyzer
AB SOLiD 3 Plus System
AB SOLiD 4 System
AB SOLiD 4hq System
AB SOLiD PI System
AB SOLiD System
AB SOLiD System 2.0
AB SOLiD System 3.0

Complete Genomics

PacBio RS
PacBio RS II
PacBio Sequel
PacBio Sequel II

Ion Torrent PGM
Ion Torrent Proton
Ion Torrent S5 XL
Ion Torrent S5

AB 310 Genetic Analyzer
AB 3130 Genetic Analyzer
AB 3130xL Genetic Analyzer
AB 3500 Genetic Analyzer
AB 3500xL Genetic Analyzer
AB 3730 Genetic Analyzer
AB 3730xL Genetic Analyzer

GridION
MinION
PromethION

BGISEQ-500
DNBSEQ-G400
DNBSEQ-T7
DNBSEQ-G50
MGISEQ-2000RS